jsonlog/jsonlog.c

```diff
@@ -193,6 +205,24 @@ setup_formatted_log_time(void)
 	memcpy(formatted_log_time + 19, msbuf, 4);
 }
 
+/*
+ * setup formatted_start_time
+ */
+static void
+setup_formatted_start_time(void)
+{
+	pg_time_t       stamp_time = (pg_time_t) MyStartTime;
+
+	/*
+	 * Note: we expect that guc.c will ensure that log_timezone is set up (at
+	 * least with a minimal GMT value) before Log_line_prefix can become
+	 * nonempty or CSV mode can be selected.
+	 */
+	pg_strftime(formatted_start_time, FORMATTED_TS_LEN,
```

Comment:
Shouldn't this generate a timestamp in the same format as setup_formatted_log_time() to have something machine-readable? Note commit f7960f1.

Reply:
Honestly I have just copy/paste the function from elog.c, I guess that we want the same output.